tests/chainerx_tests/unit_tests/routines_tests/test_normalization.py

```diff
@@ -67,57 +68,137 @@ def _create_batch_norm_ndarray_args(
 ]
 
 
-@pytest.mark.parametrize('x_shape,reduced_shape,axis', _batch_norm_params)
-@pytest.mark.parametrize('eps', [None, 3e-5, 1.2])
-@pytest.mark.parametrize('decay', [None, 0.5])
-@pytest.mark.parametrize_device(['native:0', 'cuda:0'])
-def test_batch_norm(
-        device, x_shape, reduced_shape, eps, decay, axis, float_dtype):
-    def create_args(xp):
-        return _create_batch_norm_ndarray_args(
-            xp, device, x_shape, reduced_shape, reduced_shape, reduced_shape,
-            reduced_shape, float_dtype)
+@op_utils.op_test(['native:0', 'cuda:0'])
+@chainer.testing.parameterize_pytest(
+    'x_shape,reduced_shape,axis', _batch_norm_params)
+@chainer.testing.parameterize_pytest('eps', [2e-5, 5e-1])
+@chainer.testing.parameterize_pytest('decay', [None, 0.5])
+@chainer.testing.parameterize_pytest('contiguous', [None, 'C'])
+class TestBatchNorm(op_utils.ChainerOpTest):
+
+    def setup(self, float_dtype):
+        self.dtype = float_dtype
+
+        eps = self.eps
+        decay = self.decay
+        axis = self.axis
+
+        optional_args = {}
+        if eps is not None:
+            optional_args['eps'] = eps
+        if decay is not None:
+            optional_args['decay'] = decay
+        if axis is not None:
+            optional_args['axis'] = axis
+        self.optional_args = optional_args
+
+        # - Non-contiguous running values which are updated in-place are not
+        # supported by CUDA.
+        # - Non-contiguous gamma and beta is not supported by CUDA.
+        # TODO(hvy): Support non-contiguous gamma and beta with CUDA. Create a
+        # contiguous copy in the cuDNN wrapper.
+        self.is_cuda = chainerx.get_default_device().backend.name == 'cuda'
+        if self.is_cuda and self.contiguous is None:
+            self.skip_forward_test = True
+            self.skip_backward_test = True
+            self.skip_double_backward_test = True
```

Comment:
It's better to `raise unittest.SkipTest('...')`.

Reply:
Fixed.